src/fenicsxconcrete/finite_element_problem/base_material.py

```diff
@@ -54,10 +86,11 @@ def __init__(
         self.pv_output_file = Path(pv_path) / (pv_name + ".xdmf")
 
         # setup fields for sensor output, can be defined in model
-        self.displacement = None
-        self.temperature = None
-        self.degree_of_hydration = None
-        self.q_degree_of_hydration = None
+        self.fields = None
+        # self.displacement = None
```

Comment:
Should MaterialProblem initialize the q_fields for consistency or is that not necessary/desirable?

Reply:
Yes, I added this in the TODO at the top of the page.

Reply:
Resolved in newest commit